webpack.config.js

```diff
@@ -298,9 +298,14 @@ function getWebpackConfig( { cssFilename, externalizeWordPressPackages = false }
 		// no chunks or dll here, just one big file for the desktop app
 		webpackConfig.output.filename = '[name].js';
 		webpackConfig.output.chunkFilename = '[name].js';
-	} else {
-		// jquery is only needed in the build for the desktop app
-		// see electron bug: https://github.com/atom/electron/issues/254
+	}
+
+	// jquery is only needed in the build for the desktop app
+	// see electron bug: https://github.com/atom/electron/issues/254
+	if ( calypsoEnv !== 'desktop' && ! config.isEnabled( 'gutenberg' ) ) {
```

Comment:
We shouldn't use config feature flags this way at build time. Feature flags are meant for runtime decisions, not build time ones.